src/integrations/serde.rs

```diff
@@ -1,13 +1,39 @@
 use serde::{de, ser};
 use serde::ser::SerializeMap;
-use std::collections::HashMap;
 use std::fmt;
+use std::collections::HashMap;
 
 use ::{GraphQLError, Value};
 use ast::InputValue;
 use executor::ExecutionError;
 use parser::{ParseError, Spanning, SourcePosition};
 use validation::RuleError;
+use serde_json::Value as Json;
+
+
+impl InputValue {
+    /// Converts serde_json::Value to juniper::InputValue
+    pub fn from_json(json: Json) -> InputValue {
+        match json {
+          Json::Number(num) => {
```

Comment:
Since Serde doesn't have I64 etc but just a general number it is unclear whether u64 should be converted to an i64 as well.  Please note if you want me to handle this case differently.